### PR TITLE
docs: Add Cookbook Chinese version 

### PR DESCRIPTION
**Pull request by innovation64**
# Add Cookbook Chinese version of four articles
issue #34
Hi I just translated articles into Chinese 
- Automatic Embeddings with TEI through Inference Endpoints
- Embedding multimodal data for similarity search using 🤗 transformers, 🤗 datasets and FAISS
- Fine-tuning a Code LLM on Custom Code on a single GPU
- Simple RAG for GitHub issues using Hugging Face Zephyr and LangChain
- Advanced RAG on HuggingFace documentation using LangChain
- RAG Evaluation

 @MKhalusova

### CODE REVIEW

**Comment by MKhalusova** [comment]
cc @stevhliu can you please check the translation?

**Comment by innovation64** [comment]
fine, I've already added zh-CN to the list of languages in .github/workflows/build_documentation.yml and in .github/workflows/build_pr_documentation.yml. thanks for pointing it out. @MKhalusova

**Comment by MKhalusova** [comment]
Btw, if you'd like the PR review to move a bit faster, feel free to invite another Chinese-speaking community member who could review the translation.

File: notebooks/zh/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 开源 AI 食谱
```
Comment (adeenayakup) [inline notebooks/zh/index.md]: 开源AI 实战指南 could ba a better title imo.

File: notebooks/zh/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+开源 AI 食谱是一系列 Notebook 的合集，里面展示了如何利用开源工具和模型来开发 AI 应用和解决各种机器学习问题的实际技巧和方法。
```
Comment (adeenayakup) [inline notebooks/zh/index.md]: 开源AI食谱 -> 开源AI实战指南

File: notebooks/zh/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+查看最近添加的 Notebook：
+- [通过推理端点使用 TEI 自动嵌入](automatic_embedding_tei_inference_endpoints)
+- [用 Hugging Face Zephyr 和 LangChain 针对 Github issues 构建简单的 RAG](rag_zephyr_langchain)
+- [用 🤗 transformers, 🤗 datasets 和 FAISS 嵌入多模态数据进行相似度搜索](faiss_with_hf_datasets_and_clip)
+- [在单个 GPU 上针对自定义代码微调代码 LLM](fine_tuning_code_llm_on_single_gpu)
+- [使用合成数据和 LLM 作为裁判评估 RAG](rag_evaluation)
+- [使用 LangChain 在 HuggingFace 文档上构建高级 RAG](advanced_rag)
+
+你还可以在食谱的[Github 仓库](https://github.com/huggingface/cookbook)中查看 Notebook。
```
Comment (adeenayakup) [inline notebooks/zh/index.md]: 食谱->实战指南

File: notebooks/zh/index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## 贡献
+
+开源 AI 食谱是社区和大家共同努力的成果，我们非常欢迎每个人都来参与贡献！
```
Comment (adeenayakup) [inline notebooks/zh/index.md]: 开源AI食谱 -> 开源AI实战指南

File: notebooks/zh/index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+查看食谱的[贡献指南](https://github.com/huggingface/cookbook/blob/main/README.md)了解如何添加你的“食谱”。
```
Comment (adeenayakup) [inline notebooks/zh/index.md]: 食谱->实战指南

**Review by adeenayakup — COMMENTED** [review]
Hi there! Adding some suggestions regarding the translation. Feel free to update : )

**Review by adeenayakup — APPROVED** [review]
Lgtm! Thanks for your quick update and contribution!!

**Pull request merged**